methods.md

```diff
@@ -141,7 +150,41 @@ Use this method to verify that the pin code entered by the end-user matches the
 <tr><th>Name</th><th>Data Type</th><th>Description</th><th>Mandatory</th></tr>
 <tr><td>ServiceCredentials</td><td>ServiceCredentials</td><td>See <a href="methods.md#servicecredentials">here</a> for details</td><td>Yes</td></tr>
 <tr><td>Msisdn</td><td>String</td><td>The mobile subscription number associated with the verification code.</td><td>Yes</td></tr>
-<tr><td>VerificationCode</td><td>string</td><td>The code to be verified.</td><td>Yes</td></tr>
+<tr><td>VerificationCode</td><td>String</td><td>The code to be verified.</td><td>Yes</td></tr>
+</table>
+
+### Response types
+N/A.
+
+
+## PreAuthorize
+
+PreAuthorized payments are recurring payments where the End User have confirmed that the Pay transaction for a merchant can be executed without End User confirmation on each transaction.
+
+This method returns a token to be used in subsequent Pay requests. The token is issued by Strex and does not expire. However it can be manually invalidated by Strex in some situations if the Code of Conduct is violated.
+
+Please note that the End User needs to be a user on the Strex platform for this to be successful.
```

Comment:
What happens if they are not a user on Strex platform? Will it trigger the registration flow?

Reply:
Don't know, just copied from Strex specification ... Checking with our product people as I don't have any unregistered numbers to test with.